avalon/Gemfile.local

```diff
@@ -1 +1,3 @@
 gem 'pg'
+gem 'tzinfo-data'
```

Comment:
We need to merge this PR as is because I'm afraid when someone rebuild our avalon image it might not work. I can't wait for this gem to make it into Avalon master branch.